app/src/App/App.test.tsx

```diff
@@ -1,6 +1,22 @@
 import {render, screen} from '@testing-library/react';
 import App from './App';
 
+test('Checks route paths are correct', () => {
+  render(<App />);
+
+  // Routes expected for site
+  const expectedRoutes = ['rate-my-cse', 'rate-my-cse/cse100s', 'rate-my-cse/cse300s', 
+                  'rate-my-cse/cse400s', 'rate-my-cse/cse500s']
+
+  // Get all Route elements in App
+  const routes = document.getElementsByTagName('Routes');
+
+  // Verify all existing routes match expected one
+  for (var i = 0; i < routes.length; i++) {
+    expect(expectedRoutes).toContain(routes[i].getAttribute('path'));
+  }
+});
+
 test('Checks App has correct button names and in order within the sidebar', () => {
   render(<App />);
```

Comment:
Does this test not give a warning when you do npm test?

Reply:
If it does, then just move this test for HomeLayout because this is supposed capture the side bar's buttons and its order

Reply:
It don't believe it does give me a warning, but I'll move it to Homepage.test.tsx anyway because it's now checking functionality of the HomeLayout component